client/parties/actions.ts

```diff
@@ -0,0 +1,121 @@
+import { PartyPayload, PartyUser } from '../../common/parties'
+import { BaseFetchFailure } from '../network/fetch-action-types'
+
+export type PartiesActions =
+  | InviteToPartyBegin
+  | InviteToPartySuccess
+  | BaseFetchFailure<'@parties/inviteToParty'>
+  | RemovePartyInviteBegin
+  | RemovePartyInviteSuccess
+  | BaseFetchFailure<'@parties/removePartyInvite'>
+  | DeclinePartyInviteBegin
+  | DeclinePartyInviteSuccess
+  | BaseFetchFailure<'@parties/declinePartyInvite'>
+  | AcceptPartyInviteBegin
+  | AcceptPartyInviteSuccess
+  | BaseFetchFailure<'@parties/acceptPartyInvite'>
+  | AddInvite
+  | RemoveInvite
+  | Init
+  | Invite
+  | Uninvite
+  | Decline
+  | Join
+  | Leave
+
+export interface InviteToPartyBegin {
+  type: '@parties/inviteToPartyBegin'
+}
+
+export interface InviteToPartySuccess {
+  type: '@parties/inviteToParty'
```

Comment:
Here too, we have zero useful information on these actions. We invited someone to a party, great. Who was it? No clue. What party was it for? Not sure.

`meta` is the typical property used when we need to provide information that the server won't return to us, and that's what you should be using for all of these.

If you want all successes to be handled through websocket publishes, fine, but I'm not sure why these useless success actions exist in that case, and you'd be better off thinking more deeply about the structure of these things instead of just copy-pasting the structure of other systems